app/models/recipe.rb

```diff
@@ -0,0 +1,5 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+  has_many :ingredients, dependent: :destroy
+  has_many :recipe_foods
```

Comment:
- [ ] Please also fix the `association` here, I encourage you to understand the ERD diagram properly and brainstorm with your partner about it.